week-3/C-array-every/exercise.js

```diff
@@ -4,15 +4,24 @@
 
 var students = ["Omar", "Austine", "Dany", "Swathi", "Lesley", "Rukmini"];
 var group = ["Austine", "Dany", "Swathi", "Daniel"];
-
-var groupIsOnlyStudents; // complete this statement
+var groupIsOnlyStudents = group.every(checkForStudents); // complete this statement
 
 if (groupIsOnlyStudents) {
   console.log("The group contains only students");
 } else {
-  console.log("The group does not contain only students");
+  console.log("The group does not contain only students");}
+
+  function checkForStudents(groupIsOnlyStudents) {
+    if (groupIsOnlyStudents.includes(students)) {
+      console.log("The group contains only students");
+    } else {
+      console.log("The group does not contain only students");
+    }
 }
 
 /* EXPECTED RESULT */
 
 // The group does not contain only studnets
+// well done Najla, expected result met
```

Comment:
Is this a note to yourself? :D

Reply:
yes it is. I need some encouragement :D

Reply:
That's cool! Positive thinking, I like it :)